presto-main/src/main/java/com/facebook/presto/operator/aggregation/state/HistogramState.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.operator.aggregation.state;
+
+import com.facebook.presto.operator.SingleChannelGroupByHash;
+import com.facebook.presto.spi.type.Type;
+
+@AccumulatorStateMetadata(stateFactoryClass = HistogramStateFactory.class, stateSerializerClass = HistogramStateSerializer.class)
+public interface HistogramState
+        extends AccumulatorState
+{
+    SingleChannelGroupByHash get();
+
+    void set(SingleChannelGroupByHash value);
+
+    void addMemoryUsage(long memory);
+
+    Type getType();
+
+    int getExpectedSizeForHashing();
```

Comment:
Every implementation is going to return a constant from this method, so just use that constant in the two places this is used